tests/integration/targets/mso_schema_site/tasks/main.yml

```diff
@@ -0,0 +1,246 @@
+  mso_schema_site:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: '{{ mso_site | default("ansible_test") }}'
+    template: Template 1
+    state: absent
+  register: rm_site_temp1
+
+- name: Remove a site from a schema with Template2
+  mso_schema_site:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: '{{ mso_site | default("ansible_test") }}'
+    template: Template 2
+    state: absent
+  register: rm_site_temp2
+
+- name: Verify rm_site_temp1 and rm_site_temp2
+  assert:
+    that:
+    - rm_site_temp1 is changed
+    - rm_site_temp1.current == {}
+    - rm_site_temp2 is changed
+    - rm_site_temp2.current == {}
+  
+- name: Remove a site from a schema with Template2 again
+  mso_schema_site:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: '{{ mso_site | default("ansible_test") }}'
+    template: Template 2
+    state: absent
+  register: rm_site_again
+
+- name: Verify rm_site_again
+  assert:
+    that:
+    - rm_site_again is not changed
+  
+# USE NON-EXISTING STATE
+- name: non_existing_state state
+  mso_schema_site:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: '{{ mso_site | default("ansible_test") }}'
+    template: Template 1
+    state: non_existing_state
+  ignore_errors: yes
+  register: non_existing_state
+
+- name: Verify non_existing_state
+  assert:
+    that:
+    - non_existing_state is not changed
+    - non_existing_state.msg == "value of state must be one of{{':'}} absent, present, query, got{{':'}} non_existing_state"
+
+# USE A NON_EXISTING_SCHEMA
+- name: non_existing_schema
+  mso_schema_site:
+    <<: *mso_info
+    schema: non_existing_schema
+    site: '{{ mso_site | default("ansible_test") }}'
+    template: Template 1
+    state: query
+  ignore_errors: yes
+  register: non_existing_schema
+
+- name: Verify non_existing_schema
+  assert:
+    that:
+    - non_existing_schema is not changed
+    - non_existing_schema.msg == "Provided schema 'non_existing_schema' does not exist"
+
+# USE A NON_EXISTING_TEMPLATE
+- name: non_existing_template
+  mso_schema_site:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: '{{ mso_site | default("ansible_test") }}'
+    template: non_existing_template
+    state: query
+  ignore_errors: yes
+  register: non_existing_template
+
+- name: Verify non_existing_template
+  assert:
+    that:
+    - non_existing_template is not changed
+    - non_existing_template.msg == "Template 'non_existing_template' not found"
+
+- name: Template attribute absent in task
+  mso_schema_site:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: '{{ mso_site | default("ansible_test") }}'
+    state: query
+  ignore_errors: yes
+  register: absent_template
+
+- name: Verify absent_template
+  assert:
+    that:
+    - absent_template is not changed
```

Comment:
can I know what is the result of this task? 
error msg or the same result as query_all or anything else?